kasa/tests/fixtures/smart/KS240.US._1.0_1.0.5.json

```diff
@@ -0,0 +1,479 @@
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "brightness",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "preset",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "on_off_gradually",
+                        "ver_code": 2
+                    },
+                    {
+                        "id": "dimmer_calibration",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "overheat_protection",
+                        "ver_code": 1
+                    }
+                ],
+                "device_id": "000000000000000000000000000000000000000001"
+            },
+            {
+                "component_list": [
+                    {
+                        "id": "device",
+                        "ver_code": 2
+                    },
+                    {
+                        "id": "firmware",
+                        "ver_code": 2
+                    },
+                    {
+                        "id": "quick_setup",
+                        "ver_code": 3
+                    },
+                    {
+                        "id": "time",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "schedule",
+                        "ver_code": 2
+                    },
+                    {
+                        "id": "countdown",
+                        "ver_code": 2
+                    },
+                    {
+                        "id": "account",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "synchronize",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "sunrise_sunset",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "cloud_connect",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "iot_cloud",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "device_local_time",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "fan_control",
+                        "ver_code": 1
+                    },
+                    {
+                        "id": "overheat_protection",
+                        "ver_code": 1
+                    }
+                ],
+                "device_id": "000000000000000000000000000000000000000000"
+            }
+        ],
+        "start_index": 0,
+        "sum": 2
+    },
+    "get_child_device_list": {
+        "child_device_list": [

```

Comment:
The fan and the dimmer are supported as separate child devices. Adding proper support will involve completing both #731 and #713, as well as converting the homeassistant integration away from the model that only powerstrips can have child devices.